jupyter_core/command.py

```diff
@@ -303,7 +303,7 @@ def main():
         sys.exit(e)
 
     try:
-        _execvp(command, sys.argv[1:])
+        _execvp(command, [command] + sys.argv[2:])
```

Comment:
Note that `command` here is an absolute path to the command (it was returned by `_jupyter_abspath`). Shouldn't we just pass the base name, i.e., the actual command, rather than the entire absolute path?

Reply:
Never mind - you're right, it *should* be the absolute path.